libs/general/lib/methods/job_icons.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+final Map<String, IconData> domains = {
```

Comment:
```suggestion
const Map<String, IconData> domains = {
```

This is technically better but there shouldn't be any real difference but I think const is also technically more efficient.